plugins/main/bridge/bridge.go

```diff
@@ -196,6 +201,7 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
+	// TODO: make this optional when IPv6 is supported
```

Comment:
After inclusion of IPv6 next line should be expanded with `&& result.IP6 == nil` so this comment is hmmm... not precise.